doc/articles/features/using-skia-gtk.md

```diff
@@ -57,9 +57,52 @@ If you want to upgrade **SkiaSharp** to a later version, you'll need to specify
 
 ```xml
 <ItemGroup>
-   <PackagReference Include="SkiaSharp" Version="2.88.0" />
-   <PackagReference Include="SkiaSharp.Harfbuzz" Version="2.88.0" />
-   <PackagReference Include="SkiaSharp.NativeAssets.Linux" Version="2.88.0" />
-   <PackageReference Update="SkiaSharp.NativeAssets.macOS" Version="2.88.0" />
+   <PackagReference Include="SkiaSharp" Version="2.88.3" />
+   <PackagReference Include="SkiaSharp.Harfbuzz" Version="2.88.3" />
+   <PackagReference Include="SkiaSharp.NativeAssets.Linux" Version="2.88.3" />
+   <PackageReference Update="SkiaSharp.NativeAssets.macOS" Version="2.88.3" />
 </ItemGroup>
 ```
+
+### .NET Native AOT support
+
+Building an Uno Platform Skia+GTK app with .NET (7+) Native AOT requires, GtkSharp 3.24.24.38 (or later), or Uno Platform 4.7 (or later).
+
+To build an app with this feature enabled:
+1. Add the following property in your `.csproj`:
+   ```xml
+   <PropertyGroup>
+   		<PublishAot>true</PublishAot>
+	</PropertyGroup>
+   ```
+1. Add the following items in your `.csproj`:
+   ```xml
+   <ItemGroup>
+		<RdXmlFile Include="MyApp.rd.xml" />
+		<RuntimeHostConfigurationOption Include="Switch.System.Reflection.Assembly.SimulatedCallingAssembly" Value="true" />
+   </ItemGroup>
+   ```
+1. Create a file name `MyApp.rd.xml`, and place it next to your csproj:
+   ```xml
+   <Directives>
+      <Application>
+         <Assembly Name="MyApp.Skia.Gtk" Dynamic="Required All" />
+         
+         <Assembly Name="SkiaSharp" Dynamic="Required All" />
+         <Assembly Name="HarfBuzzSharp" Dynamic="Required All"> />
```

Comment:
```suggestion
         <Assembly Name="HarfBuzzSharp" Dynamic="Required All" />
```

Reply:
Actually, it was enough to have just this in rd.xml file
```
<Directives>
    <Application>
        <Assembly Name="GdkSharp" Dynamic="Required All">
            <Type Name="Gdk.Screen" Dynamic="Required All"/>
        </Assembly>
        <Assembly Name="GtkSharp" Dynamic="Required All">
            <Type Name="Gtk.WindowStateEventArgs" Dynamic="Required All"/>
        </Assembly>
    </Application>
</Directives>
```

Reply:
I did test on `SkiaSharpTest.Skia.Gtk`, maybe you have slightly different setup on empty project, but can you try?

Reply:
I tested an app with your settings, and it's indeed working, thanks for the update. I'll change the docs for it.

Reply:
Yes, for the default app, but in most cases, it's easier to keep everything from the main app. That's generally the case for AOT apps using Mono, and in general, code present in the main app is relevant (unlike libraries).